packages/manager/apps/ips/src/pages/onboarding/onboardingIpOptionRow.tsx

```diff
@@ -0,0 +1,132 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { Links } from '@ovh-ux/manager-react-components';
+import { PriceDescription } from '@/components/PriceDescription/PriceDescription';
+import { useCatalogLowestPrice } from '@/data/hooks/catalog';
+import useGuideUtils from '@/pages/onboarding/useGuideUtils';
+
+interface IpOptionRowProps {
+  feature: string;
+  ipv4: string | React.ReactNode;
+  ipv6: string | React.ReactNode;
```

Comment:
```suggestion
  ipv4: React.ReactNode;
  ipv6: React.ReactNode;
```

string is included inside ReactNode